package.json

```diff
@@ -30,13 +30,15 @@
     "https": "^1.0.0",
     "libsodium-wrappers": "^0.5.4",
     "libsodium-wrappers-sumo": "^0.5.4",
-    "tsconfig-paths-webpack-plugin": "^3.2.0"
+    "tsconfig-paths-webpack-plugin": "^3.2.0",
```

Comment:
Out of curiosity, what is the purpose of this package?

Reply:
https://www.npmjs.com/package/tsconfig-paths-webpack-plugin

Reply:
Use this to load modules whose location is specified in the paths section of tsconfig.json when using webpack. This package provides the functionality of the tsconfig-paths package but as a webpack plug-in.